mobile/src/components/uam-groups-create/helpers/group-creation-get-permission-columns/group-creation-get-permission-columns.helper.ts

```diff
@@ -0,0 +1,23 @@
+import { TableColumn } from '~/common/types/types';
+import { PermissionTableAccessor } from '~/components/uam-groups-create/common/enums/enums';
+
+import { PermissionsTableRow } from '../../common/types/permissions-table-row.type';
+
+const getPermissionsColumns = (): TableColumn<PermissionsTableRow>[] => {
```

Comment:
rename folder and file to get-permissions-columns/get-permissions-columns.helper